warehouse/query-core/src/main/java/datawave/query/tables/async/event/VisitorFunction.java

```diff
@@ -215,6 +217,65 @@ public ScannerChunk apply(@Nullable ScannerChunk input) {
                         }
                     }
                     
+                    int termCount = TermCountingVisitor.countTerms(script);
```

Comment:
If we're going to run TermCountingVisitor after we've run through pushdown, then we should update it to count each pushed down "_List_" as a single term, should we not?

Reply:
more generally TermCountingVisitor should count each ivarator as a single term?

Reply:
Did you determine whether or not TermCountingVisitor counts List or FST ivarators in it's count?  I suspect it doesn't, and if it doesn't, we should update it since we're relying on it to give an accurate term count.

Reply:
I missed this comment on the last pass. It does not currently count these as terms. I will update the visitor to catch all ivarators and count them as a term each